### ISSUE TITLE
Keep Austin Moving - Mobility Week App for Austin

### ISSUE THREAD

**Issue by rubencantoo** [atxhack4change/2016-project-proposals#4]
## Short description of the idea (140 character tweet):

We need an app to deploy during Austin's Mobility Weeks where we ask citizens to participate and self-report activity. Reports help track growth and impact. 

## What is your project concept or idea? What challenge or opportunity will it solve?

We all need to contribute to the traffic congestion. This is a cultural shift, however in order for it to happen, we need people to see how much fun and easy it can be to collaborate together. We want to reduce traffic congestion on our roads and improve our air quality. 

## Who will benefit from your project? Describe the humans at the center of the problem - who are they? 
All citizens who commute around Austin, especially the core of the city will feel an impact. 
### How would they benefit? Can you tell their story?
They will have the data to coordinate their trips and help contribute to a body of data that can be tracked and studied so it can scale in other cities. Finally, it will help the people who have limited mobility access more options as demand for public services goes up over time this will help everyone use a system that meets those needs. 
### Do you have photos of them? Or quotes?

_Your answer_

### [Look at the right sidebar :arrow_upper_right: ! Do you see the Labels? Pick a few]

- Arts & Culture
- Children & Youth
- Economic Development
- Economic Opportunity
- Education
- Environment
- Food/Hunger
- Health
- Housing
**- Mobility/Transportation**
**- Neighborhoods
- Open Government/Transparency
- Public Safety
- Sustainability**
- Other

## What is the current state of your project idea? [Pick more labels :arrow_upper_right: ]

- I have a **fuzzy** area of opportunity but don’t yet fully understand the challenge
- I understand the **challenge** well but don’t yet have ideas for a solution
**- I have an idea or **concept** of a solution, but have not yet prototyped or tested it
**- I tested a **prototype** and want to build it

## Tell us more about the current state of your project idea.
We have launched 2 work from home days in 2013 and two mobility weeks in 2014 and 2015. 
### Do you have any links to text, mockups, code, or data?

no

## Tell us about yourself. 
I am the executive director or Austin+SocialGood who helped create this initiative in our city. I am an entrepreneur and a social activist. Recognized as a nominee for Change Maker of the Year for AYC 2016 and Nominated as Entrepreneur of the Year for AU40 in 2015. 
### What is your background? Why is this challenge important to you? 
I am a native Austinite, I have seen this city change, I care very much about the health and well-being of our citizens and believe that together we can collectively solve these issues when we bring the best minds together. 
### Are you representing a group in submitting this project idea? 

Austin+SocialGood

**Comment by mateoclarke** [comment]
Hi @rubencantoo! I've added some tags to your project based on your responses.

**Comment by Spunks66** [comment]
Hi @rubencantoo .. I don't know if it's appropriate to put general comments/questions, but ...

Will app include a translation function or be available in multiple languages? If the latter, what languages? If only English, why not other languages (at least Spanish)?

Austin is a diverse community, especially areas such as the Rundberg community. It's also an area with higher use of public transportation and 40+ languages spoken.

**Comment by rubencantoo** [comment]
yes, we would like to have it in English and Spanish to start with an move
on from there as we identify other languages that are in need. I can
personally translate as I am a fluent native speaker.

Ruben
ᐧ

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by ssharif1** [comment]
You're in, @rubencantoo ! We invite you to pitch your project at the 2016 ATX Hack for Change!

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email shortly regarding you speaking slot for the Saturday morning pitches. Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by danielhonker** [comment]
@rubencantoo - you should have received an invite to join Slack. If you haven't already joined the team and seen the Slack channel for your project, it's #keeptatxmoving

**Comment by rubencantoo** [comment]
Got it. Thanks.

**Comment by polunsky** [comment]
Congrats, Ruben. This should be a great project.
